bootstrap/090_post_setup.sql

```diff
@@ -69,80 +69,109 @@ CREATE OR REPLACE TASK TASKS.PROBE_MONITORING
-
-        -- Send emails
-        let emails string := act.action_taken:EMAIL;
-        if (length(emails) > 1) then
-            let subject string := 'Sundeck OpsCenter probe [{probe_name}] matched query.';
-            let body string := '
-                Query Id: {query_id}\n
-                Query User: {user_name}\n
-                Warehouse Name: {warehouse_name}\n
-                Start Time: {start_time}\n
-                Query Text: \n{query_text}
-            ';
-            let dict variant := OBJECT_CONSTRUCT('query_id', act.query_id, 'query_text', act.query_text, 'user_name', act.user_name, 'warehouse_name', act.warehouse_name, 'start_time', act.start_time, 'probe_name', act.probe_name);
-            let outcome2 string := '';
-            BEGIN
-                outcome2 := (select to_json(INTERNAL.NOTIFICATIONS(tools.templatejs(:body, :dict), tools.templatejs(:subject, :dict), 'email', :emails)));
-            EXCEPTION
-                when EXPRESSION_ERROR THEN
-                    outcome2 := SQLERRM;
-                WHEN other THEN
-                    outcome2 := SQLERRM;
-            END;
-            outcome := outcome || outcome2;
-        end if;
-
-        -- Send slack messages
-        let slackDests string := act.action_taken:SLACK;
-        IF (length(slackDests) > 1) THEN
-            let slackResult string := '';
-            let dict variant := OBJECT_CONSTRUCT('bt', CHAR(UNICODE('\u0060')), 'query_id', act.query_id, 'query_text', act.query_text, 'user_name', act.user_name, 'warehouse_name', act.warehouse_name, 'start_time', act.start_time, 'probe_name', act.probe_name);
-            let message string := '
-Sundeck OpsCenter probe [{probe_name}] matched query.\n
-Query Id: {bt}{query_id}{bt}\n
-Query User: {user_name}\n
-Warehouse Name: {bt}{warehouse_name}{bt}\n
-Start Time: {bt}{start_time}{bt}\n
-Query Text: {bt}{bt}{bt}{query_text}{bt}{bt}{bt}
-            ';
-           BEGIN
-               slackResult := (select to_json(INTERNAL.NOTIFICATIONS(tools.templatejs(:message, :dict), 'unused', 'slack', :slackDests)));
-           EXCEPTION
-           WHEN other THEN
-               slackResult := SQLERRM;
-           END;
-           outcome := outcome || slackResult;
-       END IF;
-
-
-        let name string := act.PROBE_NAME;
-        let action string := act.ACTION_TAKENS;
-        let query_id string := act.QUERY_ID;
-        insert into internal.probe_actions select CURRENT_TIMESTAMP(), :name, :query_id, parse_json(:action), :outcome;
-    END FOR;
-EXCEPTION
-    WHEN OTHER THEN
-        SYSTEM$LOG_ERROR(OBJECT_CONSTRUCT('error', 'Exception occurred during probe monitoring.', 'outcome', :outcome, 'probe_action' :action, 'SQLCODE', :sqlcode, 'SQLERRM', :sqlerrm, 'SQLSTATE', :sqlstate));
-        insert into internal.probe_actions select CURRENT_TIMESTAMP(), :name, :query_id, parse_json(:action), :outcome;
+    SYSTEM$LOG_DEBUG('probe_monitoring task beginning');
+    let outcome string := '';
```

Comment:
Lifted `outcome` out of the for loop because the `EXCEPTION` statement tried to reference it. See bigger problem below.